### ISSUE TITLE
`curvenote check` does not allow checking on any kind, or collection

### ISSUE THREAD

**Issue by stevejpurves** [curvenote/curvenote#539]
Currently the CLI for checks i limited to:

```
% curvenote check scipy --help
Usage: curvenote check [options] [venue]

Run checks from plugins or example checks on your MyST project

Arguments:
  venue            Venue to check the submission against locally

Options:
  --kind <string>  Submit to the venue using this submission kind
  -h, --help       display help for command
```

and when you run `curvenote check` only the default submission kind in checked.

## Proposal
This should be extended to accommodate multiple kinds and collections. i.e.

when running `curvenote check venue`
1. in a site where there is only 1 submission kind, just run the checks
1. where there are multiple submission kinds allow the user to select the kind
1. if there is more than 1 kind AND at last one open collection allow the user to choose the kind in the context of the collection somehow, maybe just by listing `Kind (collectionA, collectionB)`?
1. a user should be able to check against a kind even if not available in an open collection

add command line parameters:
1. `curvenote check venue --kind <kind>`
1. `curvenote check venue --collection <collection>`
   - if more than on kind associated with that collection, prompt the user
   
Open to suggestions on the above proposal too.

**Comment by stevejpurves** [comment]
note a good test case is to point to `staging` and `scipy` which currently has multiple kinds and collections.

**Comment by fwkoch** [comment]
This makes sense, feels like it should basically just follow the same interface as `submit` (just more permissive on closed collections, etc).

**Comment by stevejpurves** [comment]
fixed by #546
